cloud/passportauth/example/public/login.html

```diff
@@ -0,0 +1,73 @@
+<html xmlns="http://www.w3.org/1999/xhtml" class="">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <meta name="robots" content="noindex, nofollow">
+
+  <title>        Log in to WFM
+  </title>
+  <link href="style.css" rel="stylesheet" />
+</head>
+
+<body class="">
+<div id="kc-logo"><a href="#"><div id="kc-logo-wrapper"></div></a></div>
+
+<div id="kc-container" class="">
+  <div id="kc-container-wrapper" class="">
+
+    <div id="kc-header" class="">
+      <div id="kc-header-wrapper" class="">        Feedhenry WFM
+      </div>
+    </div>
+
+
+    <div id="kc-content" class="">
+      <div id="kc-content-wrapper" class="">
+
+
+        <div id="kc-form" class="">
+          <div id="kc-form-wrapper" class="">
+            <!-- TODO some login route -->
+            <form id="kc-form-login" class="" action="/login" method="post">
+              <div class="">
+                <div class="">
+                  <label for="username" class="">Username or email</label>
+                </div>
+
+                <div class="">
+                  <input id="username" class="" name="username" value="" type="text" autofocus autocomplete="off" />
+                </div>
+              </div>
+
+              <div class="">
+                <div class="">
+                  <label for="password" class="">Password</label>
+                </div>
+
+                <div class="">
+                  <input id="password" class="" name="password" type="password" autocomplete="off" />
```

Comment:
`autocomplete="user-password"` - TL;DR - Some browsers/versions ignore the `off` value.